d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -53,62 +54,51 @@ def on_finish(self, finish_info):
         self.is_finished = True
 
 
-import os
-simulation_id = os.environ["API_CLIENT_SIMULATION_ID"]
-domain_name = os.environ["API_CLIENT_DOMAIN_NAME"]
-websocket_domain_name = os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"]
+if os.environ['JSON_FILE_PATH'] is not None:
+    with open(os.environ['JSON_FILE_PATH']) as json_file:
+        simulation_info = json.load(json_file)
+        simulation_id = simulation_info['uuid']
+        domain_name = simulation_info['domain_name']
+        websockets_domain_name = simulation_info['web_socket_domain_name']
+else:
+    simulation_id = os.environ["API_CLIENT_SIMULATION_ID"]
+    domain_name = os.environ["API_CLIENT_DOMAIN_NAME"]
+    websockets_domain_name = os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"]
```

Comment:
I see this is used again, maybe we can create a function that sets these,  a function that returns tuple maybe
```python
simulation_id, domain_name, websockets_domain_name = some_function()
```

Reply:
If the number of variables may scale to more than 3, we should use some other way